flow/designs/ihp-sg13g2/aes/rules-base.json

```diff
@@ -93,7 +93,7 @@
         "compare": "<="
     },
     "detailedroute__route__drc_errors": {
-        "value": 0,
+        "value": 4,
```

Comment:
@maliberty I had to move the PDN from M5+TM1 to TM1+TM2. Since IHP changed the routing direction, M5 had the same orientation as M1 and they can't be connected.

Some designs now have `route DRC errors`. I assume they need to be fixed, right?

Reply:
I think so as it routed cleanly before.  What do they look like?

Reply:
<img width="1263" height="892" alt="image" src="https://github.com/user-attachments/assets/fa14a318-ab04-4eab-b1a1-c29d709a0d7f" />

The pin placer is completely ignoring those vias between Metal1 and the power ring. I will try to move the power ring a  little bit away from the edge.